### ISSUE TITLE
Bigger clock

### ISSUE THREAD

**Issue by jonaskello** [firebirdberlin/nightdream#187]
I like this app and it works well but the clock is not big enough for me. I can zoom it with pinch but even when zoomed in max there is still plenty of space to make it bigger. Ideally it would be possible to use the whole screen for just the clock. Some other apps manage to have it bigger while still showing alarms etc, [this one](https://play.google.com/store/apps/details?id=com.cama.app.huge80sclock) for example. The reason I want it as big as possible is that my eyesight is bad and when I wake up I don't want to put om my glasses to see the time. Is there a reason the clock cannot be zoomed further? See also [this review](https://play.google.com/store/apps/details?id=com.cama.app.huge80sclock&reviewId=gp%3AAOqpTOEI3RmQhhfpayVN94T0zri0mHFyODuDKoE4l0SQaDyQiOY8zqfS_xAD7r6sxoLpxiaOu2BeA6um5O0puA) to understand why it would be helpful for others.

**Comment by martin3000** [comment]
Did you zoom in landscape or portrait mode?

**Comment by firebirdberlin** [comment]
The top row and the bottom row are reserved areas which are used for the battery indicator and the alarm time panel (or the radio station buttons). I doubt that the font size will increase even if these reserved areas would be used for the clock as well. 

The digital clock style with a monospaced font (e.g. 14 segments) in landscape mode result in a clock which covers almost the entire display (except these reserved areas)

**Comment by jonaskello** [comment]
@martin3000 In landscape mode

@firebirdberlin The font does matter a lot here as you say. I was using `Roboto Regular` as I found that to be most legible. This is how it looks with this font:

![image](https://user-images.githubusercontent.com/664076/91091314-b36c0c00-e656-11ea-9bd4-36ba9a3f8fc3.png)

As you can see there is a lot of black space around the numbers above. I then switched font to `DSeg14classic` and this is how it looks now:

![image](https://user-images.githubusercontent.com/664076/91091672-3d1bd980-e657-11ea-97c8-a3715cb33dcf.png)

This is much better and it covers almost the full screen. I guess there is a need to have some space to move around the numbers to avoid burn-in.

However I don't find the digital fonts as legible as the one I was using first. I guess my follow up request would be to make the (to me) more legible `Roboto Regular` font cover the whole screen like the segment fonts can.

**Comment by jonaskello** [comment]
So now that I realized some fonts can larger I tried for one day with `DSeg14classic` zoomed in to the max. It actually works OK for me to see it without glasses. Also it does not have as many lit pixels as some of the other fonts so it does not light up a dark room as much which is a plus.

Of course it would be nice to be able to zoom all fonts to cover the full screen but at this point I feel that is just nitpicking so I'll go head and close this. Thanks for a great and useful app :-).
